### PR TITLE
Open API Update - Sync with Open API PR 243

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Issue(s)
NA

## Description
Sync with Open API Update - https://github.com/okta/openapi/pull/243.

## Category

- [ ] Bugfix
- [ ] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [ ] Documentation

### CODE REVIEW

File: src/swagger/api.yaml [hunk 1]
```diff
@@ -7546,6 +7546,51 @@ definitions:
...
+        items:
+          type: string
+        type: array
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+    x-okta-tags:
+      - Role
```
Comment (bdemers) [inline src/swagger/api.yaml:7586]: Is this right? `Role`?

Comment (bdemers) [reply src/swagger/api.yaml:7586]: The corresponding method is tagged as `user`

File: src/swagger/api.yaml [hunk 2]
```diff
@@ -7546,6 +7546,51 @@ definitions:
...
+  CatalogApplicationStatus:
+    enum:
+      - ACTIVE
+      - INACTIVE
+    type: string
+    x-okta-tags:
+      - Role
```
Comment (bdemers) [inline src/swagger/api.yaml:7593]: Same with this

File: src/swagger/api.yaml (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply src/swagger/api.yaml:7586]: cc: @bretterer

File: src/swagger/api.yaml (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply src/swagger/api.yaml:7593]: cc: @bretterer

File: src/swagger/api.yaml (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply src/swagger/api.yaml:7586]: @bdemers raised this question with @bretterer in his original PR. 

Here is his feedback - https://github.com/okta/openapi/pull/243#discussion_r440904222

Comment (bretterer) [reply src/swagger/api.yaml:7586]: This is an Administrator Role model, which is why its tagged as `Role`

Comment (bdemers) [reply src/swagger/api.yaml:7586]: `CatalogApplication` is an Administrator Role?

Oh I see now, it's from:
> `/api/v1/groups/{groupId}/roles/{roleId}/targets/catalog/apps`

I think this object might get confused with the other `*Application` objects.  But... tagging it with `role` (which will namespace it in the Java SDK), should remove that confusion

I withdraw my objection 🧑‍⚖️

Comment (arvindkrishnakumar-okta) [reply src/swagger/api.yaml:7586]: Thanks @bretterer & @bdemers !

**Pull request merged**